ADOPTERS.md

```diff
@@ -1,19 +1,22 @@
 - [SolarWinds](https://solarwinds.com)
 - Sudeep Batra ([Ericsson](https://www.ericsson.com))
 - [Metabob](https://metabob.com)
-  - [Sako WS](https://twitter.com/sakows): "We find Meshery to be the perfect to tool to help us successfully operate our service mesh. And with so many new features coming out, it's only getter better and better."
+  - [Sako WS](https://twitter.com/sakows): "We find Meshery to be the perfect tool to help us successfully operate our service mesh. And with so many new features coming out, it's only getting better and better."
 - [Asteria Aerospace](https://asteria.co.in)
-  - "As we are a very modern technology company, we always want to use the cutting edge, open source technology to excel. All our microservices are modern in nature and so is their infrastructure. We are cloud agnostic and at the same time, we are fascinated by service meshes to empower us with secure, reliable, intelligent infrastructure. When we are in an Istio world, the need for managing the mesh was always been a gap. With **Meshery**, we believe we can nurture our service mesh to perform better and more efficiently."
+  - "As we are a very modern technology company, we always want to use cutting edge, open source technology to excel. All our microservices are modern in nature and so is their infrastructure. We are cloud agnostic and at the same time, we are fascinated by service meshes to empower us with secure, reliable, intelligent infrastructure. When we are in an Istio world, the need for managing the mesh was always been a gap. With **Meshery**, we believe we can nurture our service mesh to perform better and more efficiently."
 - [BootLabs](https://www.bootlabs.in/)
-  - "We are a true Boutique Tech Consulting partner who understands the pain points of all-sized company in terms of embracing Cloud.  Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers ,the cutting edge technology and the trend is ServiceMesh now. We enable our Client to make their Micoservices more secure with Mesh. When we talk about ServiceMeshes, Meshery comes handy. We have seen the imact meshery has done for the Mesh deployment and Management and reduced the time to market "
+  - "We are a true Boutique Tech Consulting partner who understands the pain points of the all-sized company in terms of embracing Cloud. Being obsessed about anything “CLOUD” and automation first, we drive our partner to success. We always strive to provide our customers, the cutting-edge technology and the trend is Service Mesh now. We enable our clients to make their microservices more secure with Mesh. When we talk about service meshes, Meshery comes in handy. We have seen the impact Meshery has done for the Mesh deployment and Management and reduced the time to market. "
```

Comment:
Everything except Meshery and the company name should be lowercased.

Reply:
Okay, I misinterpreted the last comment, so I lowercased everything except "Meshery" and Company's name.
I was hesitant about changing "Service Mesh" as it was mentioned in uppercase while quoting the other adopters.

Reply:
That logic makes sense to me. The other quotes shouldn't have capitalized "Service Mesh" either, though.

Reply:
Same thing for stuff like "Global Scale" - that shouldn't be capitalized either.